src/features/authService/auth.service.js

```diff
@@ -15,11 +15,15 @@ export default function ($rootScope, $http, $location, $sessionStorage, $base64,
         }).then(function (res) {
             user.id = res.data;
             const authdata = $base64.encode(user.username + ':' + user.password);
-            $sessionStorage.putObject('user', {username: user.username, id: user.id, auth: authdata});
+            if(fbAuth == true){
+                $sessionStorage.putObject('user', {username: user.username, id: user.id, auth: authdata,pass : password});
```

Comment:
to jest tylko tak tymczasowo, dla użytkownika fb bo jego hasła nie mamy, i jak bedziemy chcieli żeby sobie jakieś dane zupdatował to tam musi podac hasło i on mógłby podawać hasło z fb no a to nie to hasło :P może jakiś pomysł jak to zrobić ?